coins/strat.json

```diff
@@ -0,0 +1,75 @@
+"STRAT": {
+        "meta": {
+            "symbol": "STRAT",
+            "name": "Stratis",
+            "supply": {
+                "max": "xxx"
+            },
+            "web": {
+                "websites": [
+                    "http:\/\/www.steneum.com\/"
+                ],
+                "announcement": null,
```

Comment:
there is an announcement url in coins/meta/coinmeta.json for STRAT.   Please use all meta values from there, unless inaccurate somehow.